providers/implementations/rands/drbg.c

```diff
@@ -43,7 +43,7 @@ static const OSSL_DISPATCH *find_call(const OSSL_DISPATCH *dispatch,
 
 static int rand_drbg_restart(PROV_DRBG *drbg);
 
-int drbg_lock(void *vctx)
+int ossl_drbg_lock(void *vctx)
```

Comment:
Hmmmm...  in a previous run, the choice was `s|prov|ossl_prov_|` (see `ossl_prov_drbg_generate`), now it's `s|prov_|ossl_|`.

Would you mind staying with *one* standard?  I'd suggest that, to avoid possible clashes with stuff in `crypto/`, the non-static stuff in `providers/implementations` all get to start with `ossl_prov_`, i.e. the previous renaming standard.